source/OutfitterPanel.cpp

```diff
@@ -109,8 +109,8 @@ void OutfitterPanel::Step()
 	ShopPanel::Step();
 	ShopPanel::CheckForMissions(Mission::OUTFITTER);
 	if(GetUI()->IsTop(this) && !checkedHelp)
-		if(!DoHelp("outfitter") && !DoHelp("outfitter 2") && !DoHelp("outfitter 3"))
-			// All help messages have now been displayed.
+		if(DoHelp("outfitter") || DoHelp("outfitter 2") || DoHelp("outfitter 3") || true)
```

Comment:
Is this:
`if(someExpression OR true)`
Do you want to know what the expression inside the if condition will evaluate to?
(Hint: it's always true.)

Reply:
thats the point yes
unlimited powaaar ~~I could make it longer but its so elegant this way~~